riscv_config/warl.py

```diff
@@ -55,9 +56,12 @@ def __init__(self, node, csrname, f_msb, f_lsb, spec=None):
         self.regex_legal = re.compile('\[(.*?)\]\s*(bitmask|in|not in)\s*\[(.*?)\]')
         self.regex_dep_legal = re.compile('(?P<csrname>.*?)\s*\[(?P<indices>.*?)\]\s*(?P<op>in|not in.*?)\s*\[(?P<values>.*?)\]\s*')
         self.dependencies = node['dependency_fields']
+        self.uarch_depends = {'uarch_signals': []}
+        self.uarch_signals = uarch_signals
```

Comment:
Can be derived from the `spec` variable itself. Prevents making a non-backward compatible API change.